language/German/strings.po

```diff
@@ -2466,21 +2512,25 @@ msgid ""
 "Press %1$s to cancel\n"
 "Press %2$s to continue"
 msgstr ""
+"Überprüfung abbrechen?\n"
+"\n"
+"Für Abbruch %1$s drücken\n"
+"Zum Fortsetzen %2$s drücken"
```

Comment:
I would translate it as "Zum Abbrechen.... Zum Fortsetzen"

Reply:
Fixed all _Für Abbruch_ to _Zum Abbrechen_